report/renderer/performance-category-renderer.js

```diff
@@ -311,8 +311,9 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       ({acronym: 'All'}),
       ...filterableMetrics,
     ]);
+    const nowTs = Date.now();
```

Comment:
This defiantly deserves a comment about how radio inputs need unique ids and that this is needed to support multiple reports on the same page :)

Reply:
> defiantly

😠 defiance!